AutomatedTesting/Gem/PythonTests/prefab/Prefab_Operations.py

```diff
@@ -0,0 +1,190 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from collections import Counter
+from collections import deque
+
+from PySide2 import QtWidgets
+
+from azlmbr.entity import EntityId
+from azlmbr.math import Vector3
+from editor_python_test_tools.utils import Report
+
+import azlmbr.bus as bus
+import azlmbr.editor as editor
+import azlmbr.prefab as prefab
+import editor_python_test_tools.pyside_utils as pyside_utils
+import prefab.Prefab_Test_Utils as prefab_test_utils
+
+
+class OperationResults:
```

Comment:
You may prefer to just declare the tuples inside the function that uses them.

Reply:
Will fix it.

Reply:
Fixed.